style.css

```diff
@@ -0,0 +1,16 @@
+#main{
+  background-color: black;
+  font-family: arial, helvetica, sans-serif;
+  color:white;
+  Text-align: center;
```

Comment:
Please stick to the design as much as possible (e.g., font, colours, font family, and margins) as a mandate in the project requirement 👍